src/Feed/FeedTypeInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Feed;
+
+use App\Entity\Feed;
+use App\Entity\FeedSource;
+
+interface FeedTypeInterface
+{
+    public function getAdmin(): ?array;
```

Comment:
Based on your implementation of this function in `RssFeedType` a better name would be `getAdminFormOptions`

Reply:
Also having docblocks in the interface would be a great help :-)